### PR TITLE
first working version of PIOP1 and PIOP2 plus tests and testing datas…

### LINKED ISSUE #94: [DAT]: AOMIC PIOP1 and PIOP2
### Which dataset is it?

For the [AOMIC](https://nilab-uva.github.io/AOMIC.github.io/) dataset so far the [ID1000 datagrabber](https://github.com/juaml/junifer/blob/main/junifer/datagrabber/aomic1000.py) is implemented by @verakye. We would like to use the [PIOP1](https://openneuro.org/datasets/ds002785) and [PIOP2](https://openneuro.org/datasets/ds002790/versions/2.0.0) datasets as well through junifer.

### Implementation

It can be implemented very similarly to Vera's AOMIC1000 datagrabber. I can implement the PIOP1 and PIOP2 datagrabbers based on that.

### Dataset access restrictions

- [X] Public and open access (no registration)
- [ ] Public and open access (registration required)
- [ ] Restricted access (needs approuval)
- [ ] Available only in specific locations (Juseless, Jureca)

### Anything else to say?

_No response_

### LINKED ISSUE #96: [BUG]: DataladDataGrabber with specified datadir
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am writing this issue because it seems to be an issue not with the AOMIC datagrabber but with the underlying datalad datagrabber.

This became apparent when testing the DataladAOMIC1000 datagrabber. If I specify an existing directory for installing the datalad dataset, then subsequently the datagrabber seems to get the incorrect file (i.e. by incorrectly appending the directory before the file path.) If I do not specify an existing directory, then everything works correctly in a tmp dir.

### Expected Behavior

I would like it to work the same whether or not I specify the datadir.

### Steps To Reproduce

I installed junifer from github.

```python3
from junifer.datagrabber.aomic1000 import DataladAOMIC1000

# test is an existing folder in my working directory
dg = DataladAOMIC1000(datadir="test")

with dg:
    test_element = dg.get_elements()[0] 
    out = dg[test_element]
```

Running this will produce the following error:

```
get(impossible): test/derivatives/fmriprep/sub-0347/func/sub-0347_task-moviewatching_space-MNI152NLin2009cAsym_desc-preproc_bold.nii.gz [path does not exist]
```

The same code without specifying a datadir works correctly:
```python3
from junifer.datagrabber.aomic1000 import DataladAOMIC1000

dg = DataladAOMIC1000()

with dg:
    test_element = dg.get_elements()[0] 
    out = dg[test_element]
```

Running this produces the desired result:

```
get(ok): derivatives/fmriprep/sub-0371/func/sub-0371_task-moviewatching_space-MNI152NLin2009cAsym_desc-preproc_bold.nii.gz (file) [from s3-PUBLIC...]
```

### Environment

```markdown
Usage: junifer [OPTIONS] COMMAND [ARGS]...
Try 'junifer --help' for help.

Error: No such command 'wtf'.
```

### Relevant log output

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by LeSasse**
…ets on gin

* [x] fix #94 and #95
* [x] description of feature/fix
* [x] tests added/passed
* [x] add an entry to the [latest changes](../docs/changes/latest.inc)

I added the datagrabbers for PIOP1 and PIOP2.

### CODE REVIEW

**Comment by LeSasse** [comment]
> That's fast!!
> 
> Can you fix the coverage issue? Just add a few tests for:
> 
>     * Invalid tasks
> 
>     * task as a string (not None, not List)
> 
> 
> Then it's missing the DOC changes: `docs/builtin.rst`

Should be all done now. Check out also #96. I have not had a closer look at that problem yet, but if we identify the problem, I can add it to this pull request or we do it in a separate one.

**Comment by LeSasse** [comment]
Ah i see you just fixed it :D :rocket:

File: junifer/datagrabber/tests/test_aomic.py [hunk 1]
```diff
@@ -0,0 +1,380 @@
+"""Provide tests for aomic1000."""
```
Comment (fraimondo) [inline junifer/datagrabber/tests/test_aomic.py:1]: can you place this file in `datagrabber/aomic/tests`? And can you split it in 3 files, one for each dataset?

File: junifer/datagrabber/aomic.py [hunk 2]
```diff
@@ -0,0 +1,354 @@
+"""Provide concrete implementations for AOMIC1000 data access."""
```
Comment (fraimondo) [inline junifer/datagrabber/aomic.py:1]: The idea was to create an `aomic` module (folder) and place the 3 AOMIC datagrabber inside, each one in a different file.

**Review by fraimondo — CHANGES_REQUESTED** [review]

File: junifer/datagrabber/aomic.py (hunk 2 above)
Comment (LeSasse) [reply junifer/datagrabber/aomic.py:1]: ah ok makes sense

File: junifer/datagrabber/__init__.py [hunk 3]
```diff
@@ -10,4 +10,3 @@
 from .multiple import MultipleDataGrabber
 from .pattern import PatternDataGrabber
 from .pattern_datalad import PatternDataladDataGrabber
-from .aomic1000 import DataladAOMIC1000
```
Comment (fraimondo) [inline junifer/datagrabber/__init__.py:13]: now you need to `from .aomic import DataladAOMICID1000, DataladAOMICPIOP1, DataladAOMICPIOP2`

**Review by fraimondo — CHANGES_REQUESTED** [review]
Just that small change and we are good to go!

**Review by fraimondo — APPROVED** [review]

**Pull request merged**